#request
uint16 minSize
---
#response
Arc[] arcs
uint16 imageWidth
uint16 imageHeight
